src/plugins/git/gitUtils.js

```diff
@@ -35,6 +36,10 @@ export function localGit(repositoryPath: string): GitDriver {
       // Ignore global Git settings, for test isolation.
       GIT_CONFIG_NOSYSTEM: "1",
       GIT_ATTR_NOSYSTEM: "1",
+      // Bring over the SSH configuration so that loading private repos is possible
+      // This post has some useful information on SSH_AUTH_SOCK:
+      // http://blog.joncairns.com/2013/12/understanding-ssh-agent-and-ssh-add/
+      SSH_AUTH_SOCK: process.env.SSH_AUTH_SOCK,
```

Comment:
This adds `SSH_AUTH_SOCK=undefined` to the environment when the user
does not have this environment variable set, which is a bit unclean, but
from my testing this doesn’t look like it actually interferes with the
operation. No change requested.

Reply:
Good catch. I experimented with setting it conditionally on SSH_AUTH_SOCK being present, but it meant I would need to put in a flow any-cast, or else write out an interface for the object, so I decided to just leave it as is.